add atomic-router, home page, other pages templates

Hellomyguest opened the pull request:
add atomic-router
add home page
add other pages templates

Commit:
feat(components): add atomic-router, home page, other pages templates

META DATA: issue16

makhnatkin commented on README.md:
````diff
@@ -1,24 +1,76 @@
+[![StackShare](https://img.shields.io/badge/tech-stack-0690fa.svg?style=flat)](https://stackshare.io/stackshare/awesome-stacks)
 
-```javascript
+- #### Development environment: <br>**Vite** (https://vitejs.dev/)
````
plz merge main inside this branch

makhnatkin commented on src/App.tsx:
```diff
@@ -1,40 +1,23 @@
+    <ConfigProvider
+      theme={{
+        algorithm:
+          themeValue === "DARK" ? theme.darkAlgorithm : theme.defaultAlgorithm,
```
themeValue === themes.DARK ? theme.darkAlgorithm : theme.defaultAlgorithm,

makhnatkin commented on src/features/theme-switcher/ui/ThemeSwitcher.tsx:
```diff
@@ -0,0 +1,25 @@
+    <Space className="Space">
+      День
+      <Switch
+        checked={themeValue === "DARK"}
```
Why is this code in two prs?
You should branch off the branch again from main

Commit:
Merge develop in this branch

Commit:
feat(components): home page: links wrapped by Space

META DATA: issue16

Commit:
build: add eslint-plugin-effector to devDependencies

Add eslint-plugin-effector
add husky pre-commit hook
add lintstaged configuration
add
eslintignore

META DATA: issue16

Commit:
build: change devDependencies for eslint

META DATA: issue16

Commit:
fix: add rule to eslint

META DATA: issue16

makhnatkin commented on package.json:
```diff
@@ -10,29 +10,34 @@
     "antd": "^5.0.2",
+    "atomic-router": "^0.8.0",
+    "atomic-router-react": "^0.8.1",
+    "commitizen": "^4.2.5",
```
is really need in dependecies?
I hope it needed only in devDep
yes?

Hellomyguest replied:
done

makhnatkin commented on src/pages/home/page.css:
```diff
@@ -0,0 +1,11 @@
+.Content {
+    padding: 3.125rem;
```
use 3rem

Hellomyguest replied:
done

Commit:
fix(components): fix eslint errors

fix eslint errors
change home page styling
useinit for event in ThemeSwitcher

META DATA: issue16
